skafka/src/main/scala/com/evolutiongaming/skafka/Blocking.scala

```diff
@@ -1,14 +1,28 @@
 package com.evolutiongaming.skafka
 
+import cats.effect.{Async, ContextShift}
+import com.evolutiongaming.catshelper.FromFuture
+
 import scala.concurrent.{ExecutionContext, Future}
 
 trait Blocking[F[_]] {
-  def apply[T](f: => T): F[T]
+  def apply[A](thunk: => A): F[A]
+
+  def future[A](f: => Future[A]): F[A]
 }
 
 object Blocking {
+  def apply[F[_] : Async : ContextShift : FromFuture](blockingEC: ExecutionContext): Blocking[F] =
```

Comment:
rename blockingEC -> ec/executor ? I mean that it is clear from context that we are dealing with blocking calls here

Reply:
Done.